src/rest/QueryResults.ts

```diff
@@ -127,6 +127,10 @@ export interface IQueryResults {
    * The Coveo Query Pipeline triggers, if any were configured.
    */
   triggers: ITrigger<any>[];
+  /**
+   * The keywords selected by Reveal's Refined Query feature
```

Comment:
Maybe add a little something that explain that they are the important keywords that reveal detected in the query, when it compared the {@link IQuery.bfq} to the data available in usage analytics for a given organization.